src/pages/settings/Profile/Contacts/NewContactMethodPage.tsx

```diff
@@ -163,7 +162,7 @@ function NewContactMethodPage({route}: NewContactMethodPageProps) {
                         User.clearContactMethodErrors(addSMSDomainIfPhoneNumber(pendingContactAction?.contactMethod ?? contactMethod), 'addedLogin');
                     }}
                     onClose={() => {
-                        if (!isEmptyObject(loginData?.errorFields) && pendingContactAction?.contactMethod) {
+                        if (pendingContactAction?.contactMethod) {
```

Comment:
Is it safe to not check `loginData?.errorFields` here @mkzie2? I tested it and it does solve our problem, but I was thinking about Shahidullah-Muffakir's [proposal](https://github.com/Expensify/App/issues/53819#issuecomment-2531607888), do you think it makes sense? I find it strange that I can reproduce the issue only if I close the error manually, otherwise everything happens normally.

Reply:
I see, that makes sense, thanks for clarifying. Finishing the checklist.